x-pack/platform/plugins/shared/stack_connectors/server/connector_types/slack_api/index.test.ts

```diff
@@ -79,7 +79,7 @@ describe('validate params', () => {
     expect(() => {
       validateParams(connectorType, {}, { configurationUtilities });
     }).toThrowErrorMatchingInlineSnapshot(
-      `"error validating action params: Cannot destructure property 'Symbol(Symbol.iterator)' of 'undefined' as it is undefined."`
+      `"error validating action params: undefined is not iterable (cannot read property Symbol(Symbol.iterator))"`
```

Comment:
I'm super-curious how this changed!  And nothing else related?  This is just a rando change?  Maybe the upgrade to node change?  But then why wouldn't have been caught then?  Something seems off ...

Reply:
Thanks for catching that! Reverted the change here [`f1b1920` (#220273)](https://github.com/elastic/kibana/pull/220273/commits/f1b1920411348c90460b6be4fdee410a6eba7032)